src/code/z_lifemeter.c

```diff
@@ -1,59 +1,147 @@
 #include "global.h"
 
-extern s16 D_8011FF10[3];
-extern s16 D_8011FF24[3];
-extern s16 D_8011FF38[3];
-extern s16 D_8011FF4C[3];
-extern s16 D_8011FF60[3];
-extern s16 D_8011FF74[3];
-extern s16 D_8011FF88[3];
-extern s16 D_8011FF9C[3];
-extern u8* D_8011FFB0[];
-extern u8* D_8011FFF0[];
-
-s16 D_8015FDC0[3];
-s16 D_8015FDC8[3];
-s16 D_8015FDD0[6];
-s16 D_8015FDE0[6];
-
-void Health_InitData(GlobalContext* globalCtx) {
+/***
+ * These are the colors for the hearts in the interface. The prim color is the red color of the heart
+ * for the base hearts, while the prim color for the double defense hearts is the white outline. The
+ * env color for the base hearts is the purple-ish outline, while the env color for the double defense
+ * hearts is the red color of the hearts.
+ ***/
```

Comment:
It's fairly minor, but the style for comment blocks usually starts with `/**` and ends with `*/`